src/dtype.rs

```diff
@@ -256,7 +255,7 @@ impl PyArrayDescr {
     /// Equivalent to [`numpy.dtype.flags`][dtype-flags].
     ///
     /// [dtype-flags]: https://numpy.org/doc/stable/reference/generated/numpy.dtype.flags.html
-    pub fn flags(&self) -> c_char {
+    pub fn flags(&self) -> u64 {
```

Comment:
Please expand the documentation to explain the type differences between NumPy 1.x and 2.x.

Reply:
Added a brief note in #442 so indicate the flags field was expanded. Not sure if you wanted more than that.